SwanCustomEnvironments/swancustomenvironments/scripts/builders/conda.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Check if the dependency is already in the requirements file, if not, add it to the file
```

Comment:
Why having this double block? Can't we just install our enforced version of ipykernel **after** installing the packages from the requirements file?

Reply:
I initially tried that but you can really tell the difference between both approaches. I know it aint pretty but becomes much faster, so I went for that.

Reply:
Why is it faster? Because it does another round of resolving dependencies?
I think it is still better code-wise to be consistent with the other builders, where we first install requirements and later we install ipykernel with our fixed version.